bool resultHMI
---
int64 codeHMI

